sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/flow/FlowRuleManager.java

```diff
@@ -61,7 +62,7 @@
         currentProperty.addListener(LISTENER);
         startMetricTimerListener();
     }
-
+    
```

Comment:
I think there's no need to reorder the imports and introduce extra spaces.

Reply:
Yes, I will remove the extra space.

Reply:
Oh, I see the latest change on spaces. But how about the reordering on the imports?